src/v2/Apps/Partner/Components/NavigationTabs/index.tsx

```diff
@@ -1,15 +1,36 @@
+import { breakpoints } from "@artsy/palette"
 import React from "react"
 import { createFragmentContainer, graphql } from "react-relay"
+import styled, { css } from "styled-components"
+import { HorizontalPadding } from "v2/Apps/Components/HorizontalPadding"
+import { FullBleed } from "v2/Components/FullBleed"
 import { RouteTab, RouteTabs } from "v2/Components/RouteTabs"
+import { BaseContainer, StickyContainer } from "v2/Components/StickyContainer"
 import { NavigationTabs_partner } from "v2/__generated__/NavigationTabs_partner.graphql"
 
 interface NavigationTabsProps {
   partner: NavigationTabs_partner
 }
 
+export const Container = styled(BaseContainer)`
+  ${({ stuck }) =>
+    stuck
+      ? css`
+          & > div {
+            box-shadow: 0px 0px 1px rgba(0, 0, 0, 0.08),
+              0px 2px 8px rgba(0, 0, 0, 0.12);
```

Comment:
Hm, I realize this one is a bit different but we should have consistent shadows throughout. I need to export this one from Palette: https://github.com/artsy/palette/blob/19689c4fec8de04bc0d7222ae034a4f976fadf88/packages/palette/src/helpers/shadow.ts#L1 (This is fine for now but maybe add a TODO comment about replacing it)